### PR TITLE
CLDR-13963 show placeholders in info panel

### PR DESCRIPTION
**Pull request by srl295**
- also add to Row payload
- also add to ("new") Vote API

CLDR-13963

Image example:

image

- [X] This PR completes the ticket.

(though, we can tweak the style if desired, in this or a future PR)

### CODE REVIEW

File: tools/cldr-apps/src/main/java/org/unicode/cldr/web/DataSection.java [hunk 1]
```diff
@@ -1052,6 +1053,19 @@ public String toJSONString() throws JSONException {
...
+                if (placeholderInfoMap != null && !placeholderInfoMap.isEmpty()) {
+                    JSONObject placeholderInfo = new JSONObject();
+                    for (final Map.Entry<String, PatternPlaceholders.PlaceholderInfo> e : placeholderInfoMap.entrySet()) {
+                        JSONObject subInfo = new JSONObject();
+                        subInfo.put("name", e.getValue().name);
+                        subInfo.put("example", e.getValue().example);
+                        placeholderInfo.put(e.getKey(), subInfo);
+                    }
+                    jo.put("placeholderInfo", placeholderInfo);
+                }
```
Comment (btangmu) [inline tools/cldr-apps/src/main/java/org/unicode/cldr/web/DataSection.java:1068]: could use a subroutine

**Review by btangmu — APPROVED** [review]

**Review by macchiati — APPROVED** [review]

File: tools/cldr-apps/src/main/java/org/unicode/cldr/web/DataSection.java (hunk 1 above)
Comment (btangmu) [reply tools/cldr-apps/src/main/java/org/unicode/cldr/web/DataSection.java:1068]: > the jax-rs version

?

Comment (btangmu) [reply tools/cldr-apps/src/main/java/org/unicode/cldr/web/DataSection.java:1068]: somewhere in api?

Comment (srl295) [reply tools/cldr-apps/src/main/java/org/unicode/cldr/web/DataSection.java:1068]: https://github.com/unicode-org/cldr/pull/2012/files#diff-6f10ee486b924aa8fdfd98c3dede317f9c9812b33e0dc58b7ef12b4c4e343ca8R195-R197

Comment (srl295) [reply tools/cldr-apps/src/main/java/org/unicode/cldr/web/DataSection.java:1068]: VoteAPIHelper

```java
        PatternPlaceholders placeholders = PatternPlaceholders.getInstance();
         row.placeholderStatus = placeholders.getStatus(xpath);
         row.placeholderInfo = placeholders.get(xpath);
```

jax-rs can serialize these by itself.

**Pull request merged**